project/src/test/java/com/google/sps/EventSpannerTasksTest.java

```diff
@@ -179,4 +181,33 @@ public void testEventCreationDoGetInvalid() throws Exception {
 
     Assert.assertEquals(response.getStatus(), HttpServletResponse.SC_NOT_FOUND);
   }
+
+  /**
+   * Verify registering for event that already has attendee
+   */
+  @Test
+  public void testEventRegistrationDoGet() throws Exception {
+    //Existing event to sign up for
+    SpannerTasks.insertOrUpdateUser(ATTENDEE);
+    Event event =
+        new Event.Builder(EVENT_NAME, DESCRIPTION, LABELS, LOCATION, DATE, TIME, HOST)
+        .setAttendees(new HashSet<User>(Arrays.asList(ATTENDEE))).build();
+    request.addParameter("eventId", event.getId());   
+    SpannerTasks.insertorUpdateEvent(event);
+    // User that is registering
+    authenticationHelper
+        .setEnvIsLoggedIn(true)
+        .setEnvEmail(EMAIL)
+        .setEnvAuthDomain("example.com");
+
+    new EventRegistrationServlet().doPost(request, response);
+    Event returnedEvent = new Gson().fromJson(response.getContentAsString(), Event.class);
```

Comment:
Add a line break here to separate out the different parts of the unit test per the best practices